website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,268 @@
+1. Wait for the codespace to finish building by waiting for the `postCreateCommand` command to complete; this can take several minutes:
+
+    <Lightbox src="/img/codespace-quickstart/postCreateCommand.png" title="Wait for postCreateCommand to complete" />
+
+    When this command completes, you can start using the codespace development environment. The terminal the command ran in will close and you will get a prompt in a brand new terminal. 
+
+1. At the terminal's prompt, you can execute any dbt command you want. For example:
+
+    ```shell
+    /workspaces/test (main) $ dbt build
+    ```
+
+    You can also use the [duckcli](https://duckdb.org/docs/api/cli/overview.html) to write SQL against the warehouse from the command line or build reports in the [Evidence](https://evidence.dev/) project provided in the `reports` directory.
+    
+    For complete information, refer to the [dbt command reference](https://docs.getdbt.com/reference/dbt-commands). Common commands are:
+    
+    - [dbt compile](/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+
+
+  </TabItem>
+
+</Tabs>
+
+
+
+
+
+
+
+## Generate a larger data set
+
+If you'd like to work with a larger selection of Jaffle Shop data, you can generate an arbitrary number of years of fictitious data from within your codespace. 
+
+1. Install the Python package called [jafgen](https://pypi.org/project/jafgen/). At the terminal's prompt, run:
+
+    ```shell
+    python -m pip install jafgen
+    ```
+
+1. When installation is done, run:
+    ```shell
+    jafgen [number of years to generate] # e.g. jafgen 6
+    ``` 
+    Replace `NUMBER_OF_YEARS` with the number of years you want to simulate. This command builds the CSV files and stores them in the `jaffle-data` folder, and is automatically sourced based on the `sources.yml` file and the [dbt-duckdb](/docs/core/connect-data-platform/duckdb-setup) adapter.
+
+As you increase the number of years, it takes exponentially more time to generate the data because the Jaffle Shop stores grow in size and number. For a good balance of data size and time to build, dbt Labs suggests a maximum of 6 years.
+## Next steps
+
+Now that you understand the basics of using dbt Core with DuckDB, consider optimizing your setup by documenting a project, committing changes and, scheduling a job. 
```

Comment:
```suggestion
Now that you've got dbt Core, DuckDB, and the Jaffle Shop data up and running, you can explore dbt's capabilities. Refer to these materials to get a better understanding of dbt projects and commands:

- The [About projects](/docs/build/projects) page guides you through the structure of a dbt project and its components.
- [dbt command reference](/reference/dbt-commands) explains the various commands available and what they do.
- [dbt Labs courses](https://courses.getdbt.com/collections) offer a variety of beginner, intermediate, and advanced learning modules designed to help you become a dbt expert. 
- Once you see the potential of dbt and what it can do for your organization, sign up for a free trial of [dbt Cloud](https://www.getdbt.com/signup). It's the fastest and easiest way to deploy dbt today!
- Check out the other [quickstart guides](/quickstarts) to begin integrating into your existing data warehouse.

Additionally, with your new understanding of the basics of using dbt Core with DuckDB, consider optimizing your setup by documenting a project, committing changes and, scheduling a job. 
```